commands/github-projects.js

```diff
@@ -0,0 +1,214 @@
+const fs = require("fs");
+
+module.exports = {
+    name: "github-projects",
+    description: "GitHub Project Functionality",
+    execute(command, message, args) {
+        // -github-projects: Selects GitHub Project Fnctionality
+        if (command === "github-projects") {
+            // TODO: Refactor into Switch Statement - see Code Freeze PR Notes
```

Comment:
Was about to comment suggesting a `switch` statement :)

Reply:
Haha, yeah, mind a note to remind myself - I use a ToDo extension. Wrote my first switch statement 😄. Had to nest if/else inside it because of the complexity but it looks much better. Thanks.